test/test_indexing.py

```diff
@@ -6,14 +6,6 @@
 from cosima_cookbook import database
 from sqlalchemy import func
 
-@pytest.fixture
-def session_db(tmpdir):
```

Comment:
Where are the tests picking up `session_db` with the removal of this fixture? I can't see it being imported from anywhere else.

Reply:
There's some pytest magic that picks it up from `conftest.py`